src/06-extensions.md

```diff
@@ -28,27 +29,30 @@ which are community-driven processes.
+| [BEP009](https://docs.google.com/document/d/1mqMLnxVdLwZjDd4ZiWFqjEAmOmfcModA_R535v3eQs0) | Positron Emission Tomography (PET)                                                                         | Melanie Ganz                                                               | Very close to completion. This includes finalizing the standard and writing the paper.                                                                                                                                                    | None.                                                                                  |
+| [BEP011](https://docs.google.com/document/d/1YG2g4UkEio4t_STIBOqYOwneLEs1emHIXbGKynx7V0Y) | The structural preprocessing derivatives                                                                   | Andrew Hoopes                                                              | Have finalized the surface-based and volumetric overlays (and stats), differentiate between discrete and probabilistic segmentation. The next step for this BEP is standardizing the universal look-up table and characterizing surfaces. | None.                                                                                  |
+| [BEP012](https://docs.google.com/document/d/16CvBwVMAs0IMhdoKmlmcm3W8254dQmNARo-7HhE-lJU) | The functional preprocessing derivatives                                                                   | Camille Maumet and Chris Markiewicz                                        |                                                                                                                                                                                                                                           | None.                                                                                  |
+| [BEP013](https://docs.google.com/document/d/1qBNQimDx6CuvHjbDvuFyBIrf2WRFUOJ-u50canWjjaw) | The resting state fMRI derivatives                                                                         | Steven Giavasis                                                            |                                                                                                                                                                                                                                           | None.                                                                                  |
```

Comment:
This one was merged into BEP012.

Reply:
I added a note of this for this update. In future updates it can be noted below the merged BEPs (e.g. BEP 6, 7, 8, 10)